x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/display_settings/example_result_detail_card.tsx

```diff
@@ -18,6 +18,11 @@ import { CustomSourceIcon } from './custom_source_icon';
 import { DisplaySettingsLogic } from './display_settings_logic';
 import { TitleField } from './title_field';
 
+const getAsLocalDateTimeString = (str: string) => {
+  const dateValue = Date.parse(str);
+  return dateValue ? new Date(dateValue).toLocaleString() : null;
+};
+
```

Comment:
Not sure if intentional, so decided to ask.
In the original PR, the file ResultCardDefaultFields.jsx was also updated. Should it be updated in Kibana or is it not needed?

Reply:
Kibana has no eweb

Reply:
🤦